lib/cli/test/fixtures/angular-cli/package.json

```diff
@@ -12,24 +12,24 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^2.3.1",
-    "@angular/compiler": "^2.3.1",
-    "@angular/core": "^2.3.1",
-    "@angular/forms": "^2.3.1",
-    "@angular/http": "^2.3.1",
-    "@angular/platform-browser": "^2.3.1",
-    "@angular/platform-browser-dynamic": "^2.3.1",
+    "@angular/common": "^5.0.0",
+    "@angular/compiler": "^5.0.0",
+    "@angular/core": "^5.0.0",
```

Comment:
Does this change mean that we don't support apps which use earlier versions of angular? We should reflect it in peerDeps than and do a version check in `detect.js`

Reply:
Before doing that, we should probably find out the minimal supported version (and use it in fixture)

Reply:
It MIGHT work 🕶

Reply:
The tests on `release/3.3` are fixed, but there's still a value in upgrading dependensies in app and example. Please just remove all the changes in CLI fixtures, and revert https://github.com/storybooks/storybook/pull/2208/commits/7bad0c0a26619b6dea99790a4070c3192698756b

Reply:
Yup, Angular itself should be backwards compatible. The minimum supported should be 2.3.1 (the one that's inside the fixtures)

Reply:
@alterx can you please take a look at the current 3.3 branch and see if you can fix the angular cli problems in there?

I don't know much about angular but the CLI test itself is breaking and throwing errors. It's preventing us from testing other things and releasing the next alpha version.

Reply:
Getting 
```
 ~/Projects/storybook   release/3.3  npm run test

> storybook@3.0.0 test /Users/carlosvegarodriguez/Projects/storybook
> ./scripts/test.js

? Select which tests to run Core & React & Vue (core), React-Native example (react-native-vanilla)
● Validation Error:

  Module <rootDir>/node_modules/react-native/jest/setup.js in the setupFiles option was not found.

  Configuration Documentation:
  https://facebook.github.io/jest/docs/configuration.html
```
When trying to run the tests

Reply:
Either don't select react-native option in test, or run bootstrap for react-native before running it

Reply:
@danielduan Are you talking about the issue I fixed yesterday?